src/content/docs/guides/i18n.mdx

```diff
@@ -117,15 +122,36 @@ leftSidebar.sponsoredBy: Sponsorisé par
 # ...
 ```
 
-See the [English](https://github.com/withastro/docs/tree/main/src/content/i18n/em.yml) or [French](https://github.com/withastro/docs/tree/main/src/content/i18n/fr.yml) files in the docs repo as examples of a what a full file looks like.
+See the [English](https://github.com/withastro/docs/tree/main/src/content/i18n/en.yml) or [French](https://github.com/withastro/docs/tree/main/src/content/i18n/fr.yml) files in the docs repo as examples of a what a full file looks like.
+
+#### Sidebar navigation labels
+
+Similar to reusable UI strings, we have labels for the groups displayed in the docs sidebar. For example, the labels for the sidebar tabs are stored in these files.
```

Comment:
I was avoiding “headings” because this includes stuff like the tabs which I’m not sure I’d describe as “headings”? I guess we can adjust your suggestion like this:
```suggestion
Similar to reusable UI strings, labels for the groups displayed in the docs sidebar are translated in their own file.
```